src/DataConverters/DefaultMysqlDataConverter.php

```diff
@@ -287,19 +289,37 @@ public static function toArray($value) {
     }
 
     /**
-     * @param mixed $value
+     * @param object $value
      * @return string
      */
-    public static function serialize($value) {
+    public static function serializeObject($value) {
         return \serialize($value);
     }
 
     /**
-     * @param mixed $value
-     * @return mixed
+     * @param string $value
+     * @return object|null
+     */
+    public static function deserializeObject($value) {
+        if (!\is_string($value)) throw new \obo\Exceptions\Exception("");
```

Comment:
Im not sure why we wanted that exception... and when

Reply:
because the notice can be turned off on the server

Reply:
implementation should also take into account the "nullable" attribute of property

Reply:
makes no sense imho